modules/drivers/microphone/respeaker.cc

```diff
@@ -28,15 +30,15 @@ void report_error(PaError err, const std::string &func_name) {
 }
 
 // Stream
-Stream::~Stream() { Pa_CloseStream(pastream_ptr_); }
+Stream::~Stream() {
+  Pa_CloseStream(pastream_ptr_);
+  free(inputParameters_ptr_);
```

Comment:
Does naming like `inputParameters_ptr_` meet our coding style ? @xiaoxq

Reply:
will fix this naming in the next commit

Reply:
> will fix this naming in the next commit

OK.

Reply:
In the last pr I was suggested to add _ptr suffix for pointer variable for clarification so... I am ok with both

Reply:
> In the last pr I was suggested to add _ptr suffix for pointer variable for clarification so... I am ok with both

Both `input_parameters_ptr_` and `inputParametersPtr` should be ok if not constraint by our coding style,
While `inputParameter_ptr` may not look good in any coding style.

Reply:
Sorry I didn't get it just now lol. You are right. Let me fix it

Reply:
Fixed